judge/migrations/0076_problem_statistics.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-11-21 00:23
+from __future__ import unicode_literals
+
+import django.core.validators
+from django.db import migrations, models
+from django.db.models import F
+
+def compute_user_count(apps, schema_editor):
+    Problem = apps.get_model('judge', 'Problem')
+    for problem in Problem.objects.using(schema_editor.connection.alias):
+        problem.user_count = problem.submission_set.filter(points__gte=F('problem__points')).values('user').distinct().count()
```

Comment:
This is all the same problem, so you might as well just use `problem.points` instead of that `F` non-sense?

Reply:
Also I think you should check `result='AC'` while you are at it, since 5.0/5 WA has happened. That is in fact how users who can see submissions are calculated.